examples/simulation/plot_stc_metrics.py

```diff
@@ -1,7 +1,7 @@
 """
 ===============================================
 Compare simulated and estimated source activity
-===============================================
+===============================================.
```

Comment:
Changes like this (messing up the title underlining) are what's breaking CircleCI